styleguide/source/assets/js/main-navigation.js

```diff
@@ -1,36 +1,87 @@
 (function ($, Drupal) {
+
   Drupal.behaviors.ama_mainNavigation = {
     attach: function (context, settings) {
 
+      var $categoryNavWrapper = $('.ama_category_navigation_wrapper');
       var $categoryNavigationMenu = $('.ama_category_navigation_menu');
       var $mobileSearchTrigger = $('.global-search-trigger');
       var $mobileSearch = $('.ama__global-search');
+      var $viewportHeight = $(window).innerHeight();
+      var $mainNav = $('.ama__main-navigation ');
+      var $categoryNavMenuHeight = $('.ama_category_navigation_menu').outerHeight();
+      var $categoryNavMenuMResizedHeight = 0;
```

Comment:
The `$` vars dictate a jQuery object. `$categoryNavMenuResizedHeight`, `$categoryNavMenuHeight`, and `$viewportHeight` are not jQuery objects, just ints. Please update the variable names.